gradle/testing/randomization.gradle

```diff
@@ -199,7 +199,12 @@ allprojects {
 
         // Enable security manager, if requested. We could move the selection of security manager and security policy
         // to each project's build/ configuration but it seems compact enough to keep it here for now.
-        if (Boolean.parseBoolean(testOptionsResolved["tests.useSecurityManager"])) {
+        def useSecurityManager = Boolean.parseBoolean(testOptionsResolved["tests.useSecurityManager"]);
```

Comment:
@HoustonPutman, I would prefer we have generalized mechanisms instead of one-off peculiarities, and we have them.  See globals.gradle propertyOrDevant and its friends.  Is that stuff sufficient?  If we need to toggle settings globally, we already have gradle.properties

Reply:
Are you talking about the `testOptionsResolved` thing? If so it's all over this randomization grade file. I have no idea why it's used but I just added the ability for a project to override whatever it is.

Reply:
I glossed over testOptionsResolved but glad you pointed this out now.  This is a better example than those I listed.  Basically, my point is why don't you go enhance randomization.gradle to compute testOptionsResolved with a new ability for a gradle module/project change tweak *whatever* test options instead of making useSecurityManager a special case.  I'm actually surprised, given all the flexibility here, that it's not _already_ possible.  I think it is actually... notice it calls propertyOrDefault which in turn consults project properties.